Core/source/test.ts

```diff
@@ -745,18 +761,22 @@ ava.default('verify plain-text signed message by providing it wrong key (fail: c
 
 ava.default('verify plain-text signed message that you edited after signing. This invalidates the signature. With correct key. (fail: signature mismatch)', async t => {
   const { keys, pubKeys } = getKeypairs('rsa1');
-  const { json: decryptJson, data: decryptData } = parseResponse(await endpoints.parseDecryptMsg({ keys, isEmail: true, verificationPubkeys: pubKeys }, [await getCompatAsset('mime-email-plain-signed-edited')]));
+  const { json: decryptJson, data: decryptData } = parseResponse(
+    await endpoints.parseDecryptMsg({ keys, isEmail: true, verificationPubkeys: pubKeys },
+      [await getCompatAsset('mime-email-plain-signed-edited')]));
   expect(decryptJson.replyType).equals('plain');
   expect(decryptJson.subject).equals('mime email plain signed');
   const parsedDecryptData = JSON.parse(decryptData.toString());
   expect(!!parsedDecryptData.verifyRes).equals(true);
-  expect(parsedDecryptData.verifyRes.match).equals(false);
+  expect(parsedDecryptData.verifyRes.match).is.null;
```

Comment:
Unsure if this is a breaking change for iOS app or not - will find out later as well.

Reply:
https://github.com/FlowCrypt/flowcrypt-ios/issues/1379